components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/username/NotificationUsernameRecoveryManager.java

```diff
@@ -198,33 +199,61 @@ private String getUsernameByClaims(UserClaim[] claims, int tenantId)
             UserClaim claim = claims[i];
             if (claim.getClaimURI() != null && claim.getClaimValue() != null) {
 
+                if (log.isDebugEnabled()) {
+                    log.debug("Searching users for " + claim.getClaimURI() + ":" + claim.getClaimValue());
```

Comment:
Better to have in following way

`"Searching users for claim: " + claim.getClaimURI() + " with the value: " + claim.getClaimValue()`